### ISSUE TITLE
Recommended inference settings to use for baseline numbers?

### ISSUE THREAD

**Issue by codezakh** [balrog-ai/BALROG#6]
Hi Balrog team! What settings should I use to closely match the settings you used for Table 2 and Table 3 of the Balrog paper? 

Would be great if you have a config file or arguments for the `eval.py` script that I can run to get some baseline numbers. I'm currently running `llama-3.1-8b-instruct` using:

```shell
 python eval.py \
  agent.type=naive \
  agent.max_image_history=0 \
  agent.max_history=16 \
  eval.num_workers=32 \
  client.client_name=vllm \
  client.model_id=meta-llama/Llama-3.1-8B-Instruct \
  client.base_url=http://0.0.0.0:8080/v1
```

I'm getting results that look like:
```
Summary of Results:
Overall Average Progression: 2.55% ± 0.74%
Per-Environment Results:
  babyai: 4.00% ± 2.77%, Episodes: 50
  babaisai: 5.83% ± 2.14%, Episodes: 120
  textworld: 2.94% ± 1.06%, Episodes: 30
  crafter: 0.00% ± 0.00%, Episodes: 10
  nle: 0.00% ± 0.00%, Episodes: 5
  minihack: 2.50% ± 2.47%, Episodes: 40
```

I haven't made any code changes. 

These look low compared to what you were able to get with `Llama-3.1-8b-instruct` in Table 2. I'm only able to get 2.55% progression compared to the progression you were able to get (14%+). 

It's also totally okay if the numbers in Table 2 depend on some features that only exist on an internal dev branch / aren't released yet. I mostly want to know what settings you recommend to get "baseline" numbers for standard open LLMs like the Llama3 and Gemma2 family, so I can test whether training is improving the model's performance or fixing some kind of mistake I'm making during inference. 😅

**Comment by DavidePaglieri** [comment]
Hi @codezakh, thank you for spotting this. We have identified a bug introduced while we were cleaning up the code for the open-sourcing, and did not notice this as we are still developing on the balrog-dev repo currently which doesn't have it. PR #7 fixes the problem. 
 
Please pull from the new updated main. And let us know, sorry for the inconvenience.

**Comment by codezakh** [comment]
Thanks @DavidePaglieri! I'll pull again and report back!

**Comment by DavidePaglieri** [comment]
Hi @codezakh, we have tested after the fix with Llama-3.1-8B-it and got average overall progression of: 13.0% ± 1.6%, which is within standard error of the evaluation on the website. I'll close the issue, however feel free to reopen if you have any more questions. Thanks again for spotting this!

**Comment by codezakh** [comment]
Just tested and it works on my end as well, thank you!
